src/pages/Topics.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { useRive, useStateMachineInput } from 'rive-react';
+import { Box, Divider, Heading } from 'theme-ui';
+
+import Seo from '../components/Seo';
+import TopicsCart from '../components/TopicsCard';
+import { isEmpty } from '../utils/isEmpty';
```

Comment:
We have this utility in `@fullstacksjs/toolbox`. It's better to use that instead.